check_sites.py

```diff
@@ -18,33 +18,48 @@
 import os
 from publicsuffix2 import PublicSuffixList
 
+def find_diff_sets(old_sets, new_sets):
+    diff_sets = {}
+    subtracted_sets = {}
+    difference = set(old_sets)^set(new_sets)
+    for diff in difference:
+        if diff in new_sets:
+            diff_sets[diff] = new_sets[diff]
+        else:
+            subtracted_sets[diff] = old_sets[diff]
+    return diff_sets, subtracted_sets
+
 
 def main():
     args = sys.argv[1:]
-    inputFile = 'first_party_sets.JSON'
-    inputPrefix = ''
-    opts, _ = getopt.getopt(args, "i:", ["data_directory="])
+    input_file = 'first_party_sets.JSON'
+    input_prefix = ''
+    with_diff = False
```

Comment:
Instead of making the default behavior to validate the whole file, should the default be to validate just the diff, with an opt-out to validate the whole file?

Reply:
In my opinion, `with_diff` should be `false` by default so that users can easily call `python3 check_sites.py` to run the checks on their local file. Since `input_prefix` is by default set to `''`, setting `with_diff` to `true` by default would result in the checks running on an empty dictionary when someone runs the checks locally.

Reply:
That sounds fine to me :+1: